### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NickHeiner**
Here are some small copy edits. I am not going to sign the Oracle Contributor Agreement for a PR this small. If that's a problem for you, you can just close this PR 😄.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## <a name="about"></a> About node-oracledb
 
-The node-oracledb add-on for Node.js powers high performance Oracle
+The node-oracledb module for Node.js powers high performance Oracle
```
Comment (cjbj) [inline README.md]: Technically its an add-on (or addon, depending on which English variant etc etc you like), see https://nodejs.org/api/addons.html

**Comment by cjbj** [comment]
Thanks for the comments: every bit helps.

File: README.md (hunk 1 above)
Comment (NickHeiner) [inline README.md]: Ah, great point! Thanks for the clarification.

**Comment by cjbj** [comment]
Thanks.  FYI if you haven't seen it already, we merge to a working repo and then push back here.

**Comment by NickHeiner** [comment]
I haven't seen that, but sounds good!

**Comment by cjbj** [comment]
Thanks for this.  It's fixed in 1.12.0-dev.  CHANGELOG is at https://github.com/oracle/node-oracledb/blob/v1.12.0-dev/CHANGELOG.md

**Pull request closed without merging**